scripts/build.js

```diff
@@ -66,7 +66,6 @@ files.forEach((filePath) => {
     const dataAsHTMLElement = getElement({ selector: 'div', htmlString: `<div>${rawData}</div>` })
     const { name } = getNameAndSize(filePath)
     const title = descriptions[pascalCase(name)]
-    if (title) dataAsHTMLElement.querySelector('svg').prepend(`<title>${title}</title>`)
```

Comment:
Need to remove it here because we are adding it when building the react / vue / elements icons